packages/next/src/client/components/static-generation-async-storage.external.ts

```diff
@@ -16,12 +20,7 @@ export interface StaticGenerationStore {
   readonly isRevalidate?: boolean
   readonly isUnstableCacheCallback?: boolean
 
-  /**
-   * If defined, this function when called will throw an error postponing
-   * rendering during the React render phase. This should not be invoked outside
-   * of the React render phase as it'll throw an error.
-   */
-  readonly postpone: ((reason: string) => never) | undefined
```

Comment:
Generally we want to avoid these methods of passing code around. Even our ComponentMod technique for exposing RSC scoped functions is really not ideal. In the rewrite to dynamic APIs references to React's postpone API are referenced directly from the modules that need them (transitively through dynamic-rendering.ts). Crucially this allows the postpone to be the "right" one for RSC vs SSR whereas in the previous implementaiton we were using the RSC postpone to postpone in SSR in some cases (this happens to work by accident of the implementation but by no means guaranteed)